.CMake/alg_support.cmake

```diff
@@ -1,7 +1,11 @@
 include(CMakeDependentOption)
 
+if(NOT OQS_KEM_DEFAULT)
```

Comment:
Could you replace this with `if(NOT DEFINED OQS_KEM_DEFAULT)`?

Reply:
Done